lib/preexec.bash

```diff
@@ -29,13 +29,13 @@ __bp_install_after_session_init
 function __check_precmd_conflict() {
 	local f
 	__bp_trim_whitespace f "${1?}"
-	! _bash-it-array-contains-element "${f}" "${precmd_functions[@]}"
+	_bash-it-array-contains-element "${f}" "${precmd_functions[@]}"
 }
 
 function __check_preexec_conflict() {
 	local f
 	__bp_trim_whitespace f "${1?}"
-	! _bash-it-array-contains-element "${f}" "${preexec_functions[@]}"
+	_bash-it-array-contains-element "${f}" "${preexec_functions[@]}"
 }
```

Comment:
It's entirely possible I reversed the conditional. 

Should it return success if there is a conflict, or if there isn't a conflict?

Reply:
we check `! __check_preexec_conflict` when we are adding, which makes sense. If the element is in the array, there is conflict, so we should not add it. It was reversed and I fixed it here

Reply:
(Now I need to go dig through my main branch to find out if I had fixed this and didn't pick it or if my preexec has been broken for months...)

Reply:
(Yep, it's been broken for a couple of weeks since I added the `__bp_trim_whitespace` and adopted `_bash-it-array-contains-element`... 😢)

Reply:
@NoahGorny, can we get the preexec fix merged asap? ♥